playbook/app/pb_kits/playbook/pb_icon/_icon.tsx

```diff
@@ -27,7 +27,7 @@ type IconProps = {
   data?: {[key: string]: string},
   fixedWidth?: boolean,
   flip?: "horizontal" | "vertical" | "both" | "none",
-  icon: string,
+  icon: any,
```

Comment:
I couldn't get `icon: string | SVGElement`to work and I tried things like `Function` but typescript didn't like it